web-console/knikubevirt/Create-vm/step-3-storage/wizard-storage.md

```diff
@@ -0,0 +1,67 @@
+# Create Virtual Machine Wizard - Storage Step
+
+The 1st screen of the storage section in the wizard flow will show the disks in a table view. The user can add a disk (this could be either a new disk or attach an existing one).
+
+![add disk default](img/Add-disk-default.png)
+
+Once the user clicks the ‘Add disk’ button, a modal will be shown and the user can choose what kind of disk he/she wishes to add and fill in the details. The default selection will be a blank disk. The user is able to choose the type of disk from the Source drop down menu:
+- ISO
+- URL
+- Container
+- Attch disk
+
+![add disk modal](img/Add-disk-modal.png)
+
+In case the user selects ISO
+
+![Modal with ISO selected](img/Add-disk-modal-ISO-selected.png)
+
+and fills in the details in the additional required input fields that are added upon the type of selection.
+
+## Pre-populating fields
+
+In the 'General' step, we allow the user to select the guest operating system. This could allow us to auto-fill certain fields (like disk bus and network interface). The auto.fill informations can be taken from the templates which could be used in the background.
```

Comment:
```suggestion
In the 'General' step, we allow the user to select the guest operating system. This could allow us to auto-fill certain fields (like disk bus and network interface).
```
I don't think we need to include info on the template here b/c this is a scenario where a template isn't used, from what I've understood from Fabian's comment.